sites/dotnet-guide/contents/tutorials/resharper-essentials/code-formatting/index.md

```diff
@@ -0,0 +1,57 @@
+---
+type: TutorialStep
+date: 2021-03-04
+title: Code Formatting
+technologies: [.net, csharp]
+topics: [editing,ide]
+products: [resharper]
+author: citizenmatt
+subtitle: Format and reformat code for consistency in your solution.
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://TODO
+---
+
+### Format and Reformat Code
+
+Code that is formatted consistently is more readable, and more enjoyable to work with. But what is good formatting?
+How to use whitespaces and blank lines to arrange and separate code blocks, should you use tabs for indents (or spaces?), and many more questions typically arise in these discussions.
+
+ReSharper comes with formatting rules that can suit every  team. There are default formatting rules based on the default Visual Studio formatting options and numerous best practices. You can configure every detail of formatting rules, and enforce it in your code base.
```

Comment:
```suggestion
ReSharper comes with formatting rules that can suit every team. There are default formatting rules based on the default Visual Studio formatting options and numerous best practices. You can configure every detail of formatting rules, and enforce it in your code base.
```

Reply:
How about instead of "enforce it" we have:

```suggestion
ReSharper comes with formatting rules that can suit every  team. There are default formatting rules based on the default Visual Studio formatting options and numerous best practices. You can configure every detail of formatting rules, and ReSharper will apply it for you, as you type, or when you invoke [Code Cleanup](https://www.jetbrains.com/help/resharper/Code_Cleanup__Index.html).
```